### PR TITLE
(maint) create CONTRIBUTING doc

### PR DESCRIPTION
**Pull request by er0ck**
[skip ci]

### CODE REVIEW

**Comment by er0ck** [comment]
@samwoods1 👀 please

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        The first line is a real life imperative statement with a ticket number
+        from our issue tracker.  The body describes the behavior without the patch,
+        why this is a problem, and how the patch fixes the problem when applied.
+
+* Make sure you have added [RSpec](http://rspec.info/) tests that exercise your new code.  These test should be located in the appropriate `rototiller/spec/` subdirectory.  The addition of new methods/classes or the addition of code paths to existing methods/classes requires additional RSpec coverage.
+  * One should **NOT USE** the deprecated `should`/`stub` methods - **USE** `expect`/`allow`. Use of deprecated RSpec methods will result in your patch being rejected.  See a nice blog post from 2013 on [RSpec's new message expectation syntax](http://teaisaweso.me/blog/2013/05/27/rspecs-new-message-expectation-syntax/).
+* Run the spec unit tests to assure nothing else was accidentally broken, using `rake test`
+  * **Bonus**: if possible ensure that `rake test` runs without failures for additional Ruby versions (1.9, 2.0, etc). Rototiller supports Ruby 1.9+, and breakage of support for other rubies will cause a patch to be rejected.
```
Comment (samwoods1) [inline CONTRIBUTING.md:32]: There has been discussion about beaker ditching support for < 2.1 which would be nice.  But we probably need to keep this here until that becomes official.

**Comment by samwoods1** [comment]
LGTM 👍

**Comment by er0ck** [comment]
z

File: CONTRIBUTING.md (hunk 1 above)
Comment (er0ck) [inline CONTRIBUTING.md:32]: good point

**Pull request merged**